README.md

```diff
@@ -118,33 +134,126 @@ python -m tsfeatures.compare_with_r --results_directory /some/path
 
 ### Sesonal data (100 Hourly M4 time series)
 
-| feature           |   diff | feature      | diff | feature   | diff    | feature    | diff    |
-|:------------------|-------:|:-------------|-----:|:----------|--------:|:-----------|--------:|
-| series_length     |   0    |seas_acf1     | 0    | trend | 2.28 | hurst | 26.02 |
-| flat_spots        |   0    |x_acf1|0| arch_r2 | 2.29 | hw_beta | 32.39 |
-| nperiods          |   0    |unitroot_kpss|0| alpha | 2.52 | trough | 35 |
-| crossing_points   |   0    |nonlinearity|0| beta | 3.67 | peak | 69 |
-| seasonal_period   |   0    |diff1_acf10|0| linearity | 3.97 |
-| lumpiness         |   0    |x_acf10|0| curvature | 4.8 |
-| stability         |   0    |seas_pacf|0| e_acf10 | 7.05 |
-| arch_lm           |   0    |unitroot_pp|0| garch_r2 | 7.32 |
-| diff2_acf1        |   0    |spike|0| hw_gamma | 7.32 |
-| diff2_acf10       |   0    |seasonal_strength|0.79| hw_alpha | 7.47 |
-| diff1_acf1        |   0    |e_acf1|1.67| garch_acf | 7.53 |
-| diff2x_pacf5      |   0    |arch_acf|2.18| entropy | 9.45 |
+| feature         | diff | feature           | diff | feature   | diff | feature |  diff |
+|:----------------|-----:|:------------------|-----:|:----------|-----:|:--------|------:|
+| series_length   |    0 | seas_acf1         |    0 | trend     | 2.28 | hurst   | 26.02 |
+| flat_spots      |    0 | x_acf1            |    0 | arch_r2   | 2.29 | hw_beta | 32.39 |
+| nperiods        |    0 | unitroot_kpss     |    0 | alpha     | 2.52 | trough  |    35 |
+| crossing_points |    0 | nonlinearity      |    0 | beta      | 3.67 | peak    |    69 |
+| seasonal_period |    0 | diff1_acf10       |    0 | linearity | 3.97 |         |       |
+| lumpiness       |    0 | x_acf10           |    0 | curvature |  4.8 |         |       |
+| stability       |    0 | seas_pacf         |    0 | e_acf10   | 7.05 |         |       |
+| arch_lm         |    0 | unitroot_pp       |    0 | garch_r2  | 7.32 |         |       |
+| diff2_acf1      |    0 | spike             |    0 | hw_gamma  | 7.32 |         |       |
+| diff2_acf10     |    0 | seasonal_strength | 0.79 | hw_alpha  | 7.47 |         |       |
+| diff1_acf1      |    0 | e_acf1            | 1.67 | garch_acf | 7.53 |         |       |
+| diff2x_pacf5    |    0 | arch_acf          | 2.18 | entropy   | 9.45 |         |       |
 
-To replicate this results use:
+[![Build](https://github.com/FedericoGarza/tsfeatures/workflows/Python%20package/badge.svg)](https://github.com/FedericoGarza/tsfeatures/tree/master)
```

Comment:
Seems like from this point onwards this is duplicated

Reply:
de-duplicated

Reply:
This is still duplicated